### DISCUSSION TITLE
Got Error: Failed to launch browser!

### DISCUSSION THREAD

**Discussion by windyliu1118** [getgauge/gauge#2091] [General]
I upgraded gauge as latest version 1.4.1, and taiko as the latest version 1.2.7.
```
Gauge version: 1.4.1
Commit Hash: a82478a

Plugins
-------
html-report (4.0.10)
js (2.3.10)
screenshot (0.0.1)
xml-report (0.2.3)
Version: 1.0.18 (Chromium: 86.0.4214.0) RELEASE
```

```
taiko@1.2.7
```

But when running the tests, got the error below:
```
Error: Failed to launch browser!
/var/lib/jenkins/workspace/testing_branch/node_modules/taiko/.local-chromium/linux-823078/chrome-linux/chrome: symbol lookup error: /var/lib/jenkins/workspace/testing_branch/node_modules/taiko/.local-chromium/linux-823078/chrome-linux/chrome: undefined symbol: xkb_state_key_get_utf32
at onClose (/var/lib/jenkins/workspace/testing_branch/node_modules/taiko/lib/browserFetcher.js:262:11)
at Interface.helper.addEventListener (/var/lib/jenkins/workspace/testing_branch/node_modules/taiko/lib/browserFetcher.js:250:52)
at Interface.emit (events.js:203:15)
at Interface.close (readline.js:397:8)
at Socket.onend (readline.js:173:10)
at Socket.emit (events.js:203:15)
at endReadableNT (_stream_readable.js:1145:12)
at process._tickCallback (internal/process/next_tick.js:63:19)
```

Seems there are conflict in the new version of Gauge and Taiko

**Comment by windyliu1118** [comment]
I am thinking maybe there is some chromnium dependencies missing. But have no idea which one....

**Comment by zabil** [comment]
Not able to replicate this. Please share exact details of your setup, where you see this error. OS versions, docker file if any.

**Reply by windyliu1118** [reply]
@zabil
i have downgrade my taiko version to 1.0.18 and now it works fine now. So I think there should be something wrong in the latest taiko version.

**Reply by zabil** [reply]
Thanks for the tip. There might be changes to dependencies while installing chrome. Please give more info about your environment and setup scripts so that it will be easier to find the issue.
